Lab-5-Higgs.ipynb

```diff
@@ -0,0 +1,602 @@
2uYuLC5o0aYLg4GDpjau8vDysWbMGly9fRmpqKrKzs6XWzJ49e+r1eQkUjF++Z88eafxyW1tbtGrVCgMHDjQ4fnliYiI2b96MixcvSmOj9+/fX2+oyrKUXZSkpCSj05aWs7MztIXe4E+NiFBsWVQyPmhf+XCfVC7cH5UP90nl8zS+IFThwebzjsHm84MX7cqH+6Ry4f6ofLhPKp+nMdis8Gc2iYiIiOjZxWCTiIiIiBTDYJOIiIiIFMNgk4iIiIgUw2CTiIiIiBTDYJOIiIiIFMNgk4iIiIgUw2CTiIiIiBTDYJOIiIiIFMNgk4iIiIgUw2CTiIiIiBTDYJOIiIiIFMNgk4iIiIgUw2CTiIiIiBTDYJOIiIiIFMNgk4iIiIgUw2CTiIiIiBTDYJOIiIiIFMNgk4iIiIgUw2CTiIiIiBTDYJOIiIiIFMNgk4iIiIgUw2CTiIiIiBTDYJOIiIiIFMNgk4iIiIgUo67oCtCT4xgSIvucGhFRQTUhIiKi5wVbNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgx6oquQFJSEnbs2IG4uDikpqYiPz8fzs7O8Pf3R3BwMBwcHPTSb9y4EX/99Rc0Gg28vLwwYMAA+Pn56ZWt1Wqxd+9eREVFITk5GXZ2dggKCsKAAQNgaWlpsC5KlU1ERET0PKrwYPPu3btIS0tDixYt4OTkBFNTUyQkJCAqKgrHjh3DokWLYG9vDwC4desWpk+fDlNTUwQHB8Pa2hoHDx7E/PnzERoaiiZNmsjKjoiIwL59+xAQEIDXXnsNN27cwL59+xAXF4cZM2bAxOTfhl0lyyYiIiJ6XlV4sNm4cWM0btxYb3rDhg0RHh6OX375BT179gQAbN68GRkZGVi4cCE8PT0BAO3bt8ekSZOwZs0aLF26FCqVCgBw/fp1REZGIiAgAB9++KFUrqurK9atW4djx46hTZs20nQlyyYiIiJ6XlXa5jcXFxcAQEZGBgAgOzsbp0+fhq+vrxQMAoClpSVeeukl3Lx5E7GxsdL0o0ePQgiB7t27y8rt1KkTLCwscPjwYWmakmUTERERPc8qTbCZm5uL+/fv4+7du/jzzz+xcuVKAIC/vz8AICEhAXl5eahXr55eXt20K1euSNNiY2OhUqng4+MjS2tubg5PT09ZWiXLJiIiInqeVfhtdJ3o6GisXbtW+uzi4oLx48ejYcOGAIDU1FQAgKOjo15e3TRdGt3/7ezsYGZmppfewcEBly5dgkajgVqtVrRsIiIioudZpYmGWrRogRo1aiA7Oxvx8fE4deoU7t+/L83Pzc0Fk9hJpAAAIABJREFUAIMBnC7o06XR/b+oYM/c3BwAkJOTA7VarWjZj4qKikJUVBQAYMGCBXB2djZYTnlQq9XA/+pjiJLLJn1qtZrbvJLhPqlcuD8qH+6Tyudp3CeVJth0cnKCk5MTACAgIAAtW7ZEaGgocnNz0bt3bymI02g0ennz8vIA/Bvo6f6fnZ1tcFm6wNHCwkKWT4myH9W5c2d07txZ+pySkmIwXXlwdnaGtlCQ/KhUBZdN+pydnRXd31R63CeVC/dH5cN9UvlU5D6pUaNGmfJVmmc2H+Xh4QEvLy/8/PPPAAzfztYxdBvc0dER9+/fl4LFwtLS0mBrayu1PCpZNhEREdHzrNIGm0BBK+HDhw8BAO7u7jAzM8M///yjl043zdvbW5rm7e0NIYTeyzq5ubmIj4+XpVWybCIiIqLnWYUHm+np6Qannz9/HgkJCahbty6Agm6ImjVrhgsXLiA+Pl5Kl52djejoaLi5ucneDm/VqhVUKhX27NkjK/fgwYPIycmR9YOpZNlEREREz7MKv9e7atUqpKenw9fXFy4uLsjLy8PVq1dx9OhRWFlZYcSIEVLaIUOG4Ny5c5g/fz66d+8OKysrHDx4EKmpqZg6darU6TpQ0FrZtWtXREZGIiwsDP7+/tIoP40aNdILCJUsm4iIiOh5pRJCiIqswLFjx/Drr78iISFBevvcxcUFTZo0QXBwsN4bV4mJidi8eTMuXrwojV/ev39/veEkgYLxy/fs2SONX25ra4tWrVph4MCBBscvV7LsoiQlJRmdtrScnZ2hfaTj+cJSIyIUWzbp44P2lQ/3SeXC/VH5cJ9UPk/jC0IVHmw+7xhsPj940a58uE8qF+6Pyof7pPJ5GoPNCn9mk4iIiIieXQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMQw2iYiIiEgxDDaJiIiISDEMNomIiIhIMeqKrgBVDiEhjkali4hIVbgmRERE9CxhyyYRERERKYbBJhEREREphsEmERERESmGwSYRERERKYbBJhEREREphsEmERERESmGwSYRERERKYbBJhEREREphsEmERERESmGwSYRERERKYbBJhEREREphsEmERERESmGwSYRERERKYbBJhEREREpRl3RFSBl9f3jkyLn5YY4PsGaEBER0fOILZtEREREpBgGm0RERESkGAabRERERKQYBptEREREpBgGm0RERESkGAabRERERKQYBptEREREpBgGm0RERESkGAabRERERKQYBptEREREpBgGm0RERESkGAabRERERKQYBptEREREpBgGm0RERESkGAabRER
```

Comment:
I enjoy the density plots. Easier to read than the histograms for their relationship.